wp_api/src/request.rs

```diff
@@ -753,6 +756,62 @@ pub fn is_valid_json(value: impl AsRef<str>) -> bool {
     serde_json::from_str::<serde_json::Value>(value.as_ref()).is_ok()
 }
 
+#[uniffi::export]
```

Comment:
Nitpick: I don't think this function needs to be exported? I presume we could change the arguments to `&Arc<...>` and avoid some `clone`s.

Reply:
All of the clones here are `Arc`s so there is no downside to cloning them.

I think having the function exported is a good thing. In fact, it'd be good to have some integration tests for these at least in Kotlin because it's now an important part of the flow.